src/consensus/vote.cpp

```diff
@@ -390,16 +388,10 @@ void VoteManager::cleanupVotes(uint64_t pbft_round) {
         ++v;
       }
 
+      std::advance(rit, 1);
```

Comment:
I am confused here. Why move advance outside if condition?

Reply:
Ok should be fixed with my latest push!  sloppy coding late at night and memories of it carrying over into Today.  ha.